ansible/MANUAL_STEPS.md

```diff
@@ -662,11 +662,9 @@ The hosts labelled jenkins-workspace are used to "execute" the coordination of J
 
 Note that not all jobs can use jenkins-workspace servers for execution, some are tied to other hosts.
 
-The jenkins-workspace hosts are setup as standard Node.js nodes but are only given the `jenkins-workspace` label. After setup, they require the following manual steps:
+The jenkins-workspace hosts are setup as standard Node.js nodes but are only given the `jenkins-workspace` label.
 
-* Download the Coverity Build Tool for Linux x64 at <https://scan.coverity.com/download> (requires a Coverity login)
-* Extract to `/var`, e.g. so the resulting directory looks like `/var/cov-analysis-linux64-2017.07/` or similar
-* Ensure that the [node-coverity-daily](https://ci.nodejs.org/job/node-daily-coverity/configure) job matches the path used in its explicit `PATH` setting
+The playbook should download and install the Coverity build tool needed for static analysis into `/var/`. The extracted build tool should end up in a directory similar to `/var/cov-analysis-linux64-2023.6.2`. This directory must match the `PATH` setting in the [node-coverity-daily](https://ci.nodejs.org/job/node-daily-coverity/configure) job. According to Synopsis the tool is usually updated twice yearly -- if it is updated the directory will change and the job will need to be updated accordingly.
```

Comment:
We should ask to run the playbook on the other machines so they also have the new version.

Reply:
Updated the wording.